webui/react/src/pages/ModelVersionDetails.tsx

```diff
@@ -96,6 +116,10 @@ const ModelVersionDetails: React.FC = () => {
     }
   }, [basePath, navigate, tab, tabKey]);
 
+  useEffect(() => {
+    fetchWorkspace();
```

Comment:
fetchWorkspace / setWorkspace can also be avoided if you use workspace ID from the new `modelVersion.model.workspaceId`

Reply:
i fetch workspace to get workspace name since `ModelItem` doesnt have workspace name.